src/components/calcite-combobox-item/calcite-combobox-item.e2e.ts

```diff
@@ -0,0 +1,10 @@
+import { hidden, renders } from "../../tests/commonTests";
+//import { SLOTS } from "./resources";
+
+describe("calcite-combobox-item", () => {
+  it("renders", async () => renders("calcite-combobox-item", { display: "flex" }));
+
+  it("honors hidden attribute", async () => hidden("calcite-combobox-item"));
+
+  //it("has slots", () => slots("calcite-combobox-item", SLOTS));
```

Comment:
@jcfranco I think we need to enhance the `slots` commonTest to be able to test the default slot. Can we do that in this PR?

Reply:
I'll take care of that today.

Reply:
Helper updated and test wired up. 🎉